python/grass/pygrass/vector/testsuite/test_vector3d.py

```diff
@@ -19,11 +19,12 @@
 
 def generate_coordinates(number, bbox=None, with_z=False):
     """Return 2 or 3 random arrays of coordinates"""
+    rng = np.random.default_rng()
```

Comment:
All the uses of numpy are in np.random. We could probably scope smaller.

Reply:
There is also: https://github.com/OSGeo/grass/blob/2d055ab4b4f6c755ab605c71c0a3b86b6de69ad1/python/grass/pygrass/vector/testsuite/test_vector3d.py#L50

Reply:
I read once from top to bottom, and a second time from bottom to top and missed it.. so it's fine!